### ISSUE TITLE
New update keeps crashing

### ISSUE THREAD

**Issue by 21jujubees** [mhss1/MyBrain#202]
**Describe the bug**
When I start up the app, it crashes almost immediately after. I only started having this problem with the newest version, the older versions never had this issue.

**Smartphone (please complete the following information):**
 - Device: OnePlus 5T
 - OS: LineageOS 20
 - App Version: 1.0.7

**Comment by mhss1** [comment]
Does it crash when you click something or open a specific page?

**Comment by mhss1** [comment]
Also, are you using the apk from the Github repo or using F-Droid?

**Comment by 21jujubees** [comment]
Hi, I was very light on specifics earlier, my bad. I figured out the issue anyway, it has something to do with the latest version of Neo Store, which is where I installed it, it's a modern looking frontend for F-Droid. I found out the latest version of Neo Store is super unstable on my phone, because a couple hours after I submitted this issue, that app started crashing the instant I open up. 

The same issue applies to when I used My Brain shortly after updating. It would crash instantly no matter what I do. The way that I fixed it was simply by backing up the user data using oandbackup, uninstalling, and then reinstalling from the F-Droid store. I didn't have time until right just now to figure out how to keep all my data. That was my biggest concern and why I opened this issue. But I think the real issue has to do with Neo Store, it probably corrupted the app when it updated yesterday for me. Thank you for your time.
